modules/getStates.js

```diff
@@ -0,0 +1,9 @@
+const setState = (tasks, checkbox, index) => {
+  if (checkbox.checked === true) {
+    tasks[index].checked = true;
+  } else {
+    tasks[index].checked = false;
+  }
```

Comment:
You have done a great job populating checkbox status 🥇 . What I kindly suggest you is to use `inline if conditions`

**Possible suggestion**
`tasks[index].checked = checkbox.checked === true ? true : false` 

This way your code will look `CLEAN`